content/pl/_index.html

```diff
@@ -2,11 +2,14 @@
 title: "Produkcyjny system zarządzania kontenerami"
 abstract: "Automatyzacja wdrażania, skalowania i zarządzania kontenerami"
 cid: home
+sitemap:
+  priority: 1.0
+
 ---
 
 {{< blocks/section id="oceanNodes" >}}
 {{% blocks/feature image="flower" %}}
-### [Kubernetes (K8s)]({{< relref "/docs/concepts/overview/what-is-kubernetes" >}}) to otwarte oprogramowanie służące do automatyzacji procesów uruchamiania, skalowania i zarządzania aplikacjami w kontenerach.
+[Kubernetes]({{< relref "/docs/concepts/overview/what-is-kubernetes" >}}), znany też jako K8s, to otwarte oprogramowanie służące do automatyzacji procesów uruchamiania, skalowania i zarządzania aplikacjami w kontenerach.
```

Comment:
```suggestion
[Kubernetes]({{< relref "/docs/concepts/overview/what-is-kubernetes" >}}) (skrót K8s), to otwarte oprogramowanie służące do automatyzacji procesów uruchamiania, skalowania i zarządzania aplikacjami w kontenerach.
```

Reply:
W oryginale jest: "Kubernetes, also known as K8s,..."

Reply:
po polsku jakoś tak kulawo... ale się nie będę upierał

Reply:
może "(potocznie K8s)"?

Reply:
No to nie jest potocznie. tylko taki skrót. ja bym zostawił Twoją wersję.

Reply:
potocznie raczej `k8s` czyli z małej

Reply:
W dokumentacji używają "K8s", wielką literą, więc tego nie będę zmieniał. Zgadzam się, że "potocznie" sugeruje małą.